hypertrace-graphql-span-processing-schema/src/main/java/org/hypertrace/graphql/spanprocessing/schema/rule/ExcludeSpanRuleDetails.java

```diff
@@ -0,0 +1,8 @@
+package org.hypertrace.graphql.spanprocessing.schema.rule;
+
+import graphql.annotations.annotationTypes.GraphQLName;
+
+@GraphQLName(ExcludeSpanRuleDetails.TYPE_NAME)
+public interface ExcludeSpanRuleDetails extends ExcludeSpanRule, ExcludeSpanRuleMetadata {
```

Comment:
Sounds like an overkill. Can we instead do the following, and get rid of details and metadata?

```
@GraphQLName(ExcludeSpanRule.TYPE_NAME)
public interface ExcludeSpanRule extends Identifiable, ExcludeSpanRuleInfo {
  String TYPE_NAME = "ExcludeSpanRule";

  String CREATION_TIME_KEY = "creationTime";
  String LAST_UPDATED_TIME_KEY = "lastUpdatedTime";

  @GraphQLField
  @GraphQLName(CREATION_TIME_KEY)
  @GraphQLNonNull
  Instant creationTime();

  @GraphQLField
  @GraphQLName(LAST_UPDATED_TIME_KEY)
  @GraphQLNonNull
  Instant lastUpdatedTime();
}
```